scripts/build_native_lib_in_clean_repo.sh

```diff
@@ -31,7 +31,7 @@ echoerr "Moving to $PROJECT_TMP_DIR"
 cd "$PROJECT_TMP_DIR"
 
 echoerr "cloning broadinstitute/${PROJECT}"
-GIT_LFS_SKIP_SMUDGE=1 git clone git@github.com:broadinstitute/${PROJECT}.git 1>2
+GIT_LFS_SKIP_SMUDGE=1 git clone https://github.com/broadinstitute/${PROJECT}.git 1>2
```

Comment:
I think I may have been able to fix it. In which case I won't merge the last commit except for the sed change.